Makefile

```diff
@@ -49,7 +49,6 @@ esp:
 quick:
 	make -f makefile.unix
 	(cd ngu/ngu_tests; make)
-	./ngu-micropython -c 'import ngu_tests.run'
```

Comment:
`make` in ngu_tests already run tests on both mpy and py - no need to run test against mpy again